karma.conf.js

```diff
@@ -9,17 +13,20 @@ module.exports = function(config) {
 
 		// frameworks to use
 		// available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-		frameworks: ['mocha'],
+		frameworks: ['mocha', 'sinon'],
 
 		plugins: [
 			'karma-mocha',
 			'karma-phantomjs-launcher',
-			'karma-webpack'
+			'karma-webpack',
+			'karma-sinon',
+			'karma-coverage'
 		],
 
+
 		// list of files / patterns to load in the browser
 		files: [
-			'https://cdn.polyfill.io/v2/polyfill.js?flags=gated',
+			'http://cdn.polyfill.io/v2/polyfill.js?features=fetch&flags=gated',
```

Comment:
this will only return the fetch polyfill, if you want the default set of polyfills plus fetch you can do `features=default,fetch`